Lib/glyphsLib/builder/user_data.py

```diff
@@ -57,9 +60,24 @@ def to_ufo_master_user_data(self, ufo, master):
 
 
 def to_ufo_glyph_user_data(self, ufo, glyph):
+    mathData = {
+        k: v for k, v in glyph.userData.items() if k.startswith(GLYPHS_MATH_PREFIX)
+    }
+    if mathData:
+        # Convert MATH userData to top-level keys and group them under the same
+        # key so that they are in a more usable/compact form.
+        if GLYPHS_MATH_EXTENDED_SHAPE_KEY in mathData:
+            ufo.lib.setdefault(GLYPHS_MATH_EXTENDED_SHAPE_KEY, []).append(glyph.name)
+        if GLYPHS_MATH_VARIANTS_KEY in mathData:
+            ufo.lib.setdefault(GLYPHS_MATH_VARIANTS_KEY, {})[glyph.name] = dict(
+                mathData[GLYPHS_MATH_VARIANTS_KEY]
+            )
+    if self.minimal:
+        return
```

Comment:
i'm ok if you want to export the math data regardless of the `minimal` flag, but why exclude the rest of the userData if self.minimal = False? What does this change have to do with the conversion of MATH data?

Reply:
Previously, this function was not called at all if in mininmal mode (see the change in [`builder/glyph.py`](https://github.com/googlefonts/glyphsLib/pull/980/commits/2195c7046b32caaa7974ef5481cf0e41e5ebb317#diff-d082d50d6170f5a514b1c741f44911a11531f1d9badf10fc91064eda9ecc9e72)), so this is keeping the existing behavior for other userData keys.

Reply:
oh I see, didn't notice that. ok